src/yunohost/tools.py

```diff
@@ -632,9 +632,31 @@ def tools_diagnosis(auth, private=False):
 
         diagnosis['private']['regen_conf'] = service_regen_conf(with_diff=True, dry_run=True)
 
+    diagnosis['security'] = {
+        "CVE-2017-5754": {
+            "name": "meltdown",
+            "vulnerable": _check_if_vulnerable_to_meltdown(),
+        }
+    }
+
     return diagnosis
 
 
+def _check_if_vulnerable_to_meltdown():
+    # script taken from https://github.com/speed47/spectre-meltdown-checker
+    # script commit id is store directly in the script
+    SCRIPT_PATH = "/usr/share/yunohost/yunohost-config/moulinette/spectre-meltdown-checker/spectre-meltdown-checker.sh"
```

Comment:
Eh.. it ends up being a really weird location ... '`yunohost-config/moulinette`' :/ I don't know why the folder 'other' ends up being 'yunohost-config/moulinette' in the first place.. If we want to keep a clean file tree, we should really consider putting this somewhere else like in `src/yunohost/vendor` or in `data/` directly .. or a `data/misc` or something :confused:.

Reply:
We need to modify the `debian/something` then if we have another location.